.gitlab-ci-check-docker-build.yml

```diff
@@ -40,7 +42,7 @@ build:
     - export COMMIT_TAG=${CI_COMMIT_REF_SLUG}_${CI_COMMIT_SHA}
   script:
     - echo "building ${CI_PROJECT_NAME} for ${SERVICE_IMAGE}"
-    - docker build -t $SERVICE_IMAGE .
+    - docker build -t $SERVICE_IMAGE ${DOCKER_DIR:-.}
```

Comment:
Should we note that the variable is optional mby? :thinking:

Reply:
Isn't the comment enough?
```
# It assumes a Dockerfile in the root of the repository. A different
# directory can be specified using DOCKER_DIR variable in build job:
```

Reply:
Yeh, sure. Me just being nitpicky.  (:

Reply:
So can you suggest how to rephrase it? Thanks